index.html

```diff
@@ -131,53 +139,26 @@
   }
 </style>
 
-<myuw-app-bar
-  role="toolbar"
-  theme-name="MyUW"
-  theme-url="https://my.wisc.edu"
-  app-name="Search Demo"
->
+<myuw-app-bar role="toolbar" theme-name="MyUW" theme-url="https://my.wisc.edu" app-name="Search Demo">
```

Comment:
Why?

The goal isn't to have the fewest lines of code possible, the goal is to communicate clearly.

This file serves as an example. In my opinion it is easier to understand what attributes `myuw-app-bar` takes when they're listed one-per-line block style, than when they're across a single line. I feel like this change make this file less effective for its to-humans communication purpose.

Totally appreciate that this styling change may be a result of a zealous autoformatter rather than an intentional human choice! But that's also the feedback I'm trying to provide here, that intentional human choices about code formatting are worthwhile, especially in example files.